scripts/scil_tractogram_convert_trk_to_hdf5.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+"""
+Save connections as TRK to HDF5.
+
+This script is useful to convert a set of connections or bundles to a single
+HDF5 file. The HDF5 file will contain a group for each input file, with the
+streamlines stored in the specified space and origin.
+
+To make a file compatible with scil_tractogram_commit.py or
+scil_connectivity_compute_matrices.py. you will have to follow this nomenclature
+for the input files:
+in_dir/
+    |-- LABEL1_LABEL1.trk
+    |-- LABEL1_LABEL2.trk
+    |-- [...]
+    |-- LABEL90_LABEL90.trk
+
```

Comment:
Didn't read the code yet. But from a user point of view reading this doc, here's a question: what happens if one is missing? Ex, if there is not file LABEL1_LABEL2.trk. Do you infer from other files all existing labels? Do you raise a warning? Will it work nonetheless?

Reply:
There is no problem with missing labels, this is why all the other scripts need a labels_list (to know what is and what is not missing).

I added: "The value of first labels should be smaller or equal to the second labels.
Connectivity scripts in scilpy only consider the upper triangular part of the
connectivity matrix."